applications/FemToDemApplication/custom_processes/extend_pressure_condition_process.cpp

```diff
@@ -12,218 +12,270 @@
 //
 
 #include "custom_processes/extend_pressure_condition_process.h"
+#include "processes/find_elements_neighbours_process.h"
 
 namespace Kratos {
 
 template <SizeType TDim>
 ExtendPressureConditionProcess<TDim>::ExtendPressureConditionProcess(
-    ModelPart &r_model_part)
-    : mr_model_part(r_model_part)
+    ModelPart& r_model_part)
+    : mrModelPart(r_model_part)
 {
 }
 
+template <>
+void ExtendPressureConditionProcess<2>::Execute()
+{
+    // We search the neighbours for the generation of line loads
+    auto find_neigh = FindElementalNeighboursProcess(mrModelPart, 2, 5);
+    find_neigh.Execute();
+    auto& r_process_info = mrModelPart.GetProcessInfo();
+
+    // Remove previous line loads-> Only the 1st iteration
+    if (r_process_info[ITER] == 1) {
```

Comment:
Huuuum, is this the non linear iteration, the step?, the NR uses NL_ITERATION_NUMBER

Reply:
nope, it is an internal variable that ensures several itartions when creating the new pressure loads